Automattic/sensei
# Add constant for disabling Action Scheduler loading

Resolves #7135

## Proposed Changes

* Introduce a constant (`SENSEI_DO_NOT_LOAD_ACTION_SCHEDULER`) for disabling the loading of the Action Scheduler library. It's the same one we use in sensei-pro.
* Don't initialize the Sensei `Action_Scheduler` class and the Migrations if the Action Scheduler library is not loaded.

## Testing Instructions
1. Add `define( 'SENSEI_DO_NOT_LOAD_ACTION_SCHEDULER', true );` to your `wp-config.php`
2. Make sure the action scheduler is not loading.

I wasn't able to find a way to add unit tests for this.

## Pre-Merge Checklist
